spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -476,6 +476,37 @@ def expect_changes_made_by(admin, log_items)
     end
   end
 
+  describe "POST #destroy_user_creations" do
+    let(:admin) { create(:admin) }
+    let(:user) { create(:user, banned: true) }
+    let(:other_owner) { create(:user, banned: false) }
+    let!(:collection1) { create(:collection) }
+    let!(:collection2) { create(:collection) }
+
+    before do
+      collection1.collection_participants = [create(:collection_participant, user: user, collection: collection1, participant_role: CollectionParticipant::OWNER)]
+      collection1.save
+      # Create the second user and make them an owner of the collection
+      create(:collection_participant, user: other_owner, collection: collection2, participant_role: CollectionParticipant::OWNER)
+      create(:collection_participant, user: user, collection: collection2, participant_role: CollectionParticipant::MEMBER)
+    end
+
+    context "when deleting the first user" do
+      it "deletes the first user's collection but preserves the second user's collection" do
+        admin.update(roles: ["policy_and_abuse"])
+        fake_login_admin(admin)
+        expect do
+          post :confirm_delete_user_creations, params: { id: user.login }  
+          post :destroy_user_creations, params: { id: user.login }
+        end.to change(Collection, :count).by(-1)
```

Comment:
I think the assertions that follow are enough, so how about simplifying this to:
```suggestion
        post :destroy_user_creations, params: { id: user.login }
```

Reply:
I like having this extra check in there, personally.